main.css

```diff
@@ -353,6 +371,21 @@ button:hover {
   grid-template-rows: auto;
 }
 
+form {
+  margin: auto;
+  justify-content: center;
+  text-align: center;
+}
```

Comment:
According to the project requirements, the form layout should be rendered using flexbox.

<img width="255" alt="Screen Shot 2022-05-31 at 8 33 42" src="https://user-images.githubusercontent.com/82426682/171185986-a2bfd3f8-328c-49d4-8e96-3218ec0d9f02.png">

Your form is using grid, kindly fix this.

Reply:
Hi Sir, I already solved it, check the next pull request